res/controllers/Hercules-DJ-Console-RMX-scripts.js

```diff
@@ -450,6 +450,19 @@ HerculesRMX.Deck.prototype.cueSelectHandler = function(value) {
    }
 };
 
+HerculesRMX.Deck.prototype.sourceSelectHandler = function(value) {
```

Comment:
On Wed, 2015-05-27 at 08:56 -0700, Owen Williams wrote:
> > +HerculesRMX.Deck.prototype.sourceSelectHandler = function(value) {
> 
> is there a reason this has to be a script binding and not just a basic toggle button?

I have no idea, I was happy that I was able to make it work.